Documentation/contributor-guide/triage_issues.md

```diff
@@ -4,43 +4,144 @@
 
 Speed up issue management.
 
-The `etcd` issues are listed at https://github.com/etcd-io/etcd/issues
-and are identified with labels. For example, an issue that is identified
-as a bug will eventually be set to label `area/bug`. New issues will
-start out without any labels, but typically `etcd` maintainers and active contributors
-add labels based on their findings. The detailed list of labels can be found at
-https://github.com/kubernetes/kubernetes/labels
-
-Following are few predetermined searches on issues for convenience:
-* [Bugs](https://github.com/etcd-io/etcd/labels/area%2Fbug)
-* [Help Wanted](https://github.com/etcd-io/etcd/labels/Help%20Wanted)
-* [Longest untriaged issues](https://github.com/etcd-io/etcd/issues?utf8=%E2%9C%93&q=is%3Aopen+sort%3Aupdated-asc+)
+The `etcd` issues are listed at <https://github.com/etcd-io/etcd/issues> and are identified with labels. For example, an issue that is identified as a bug will be set to label `type/bug`.
+
+The etcd project uses labels to indicate common attributes such as `area`, `type` and `priority` of incoming issues.
+
+New issues will often start out without any labels, but typically `etcd` maintainers, reviewers and members will add labels by following these triage guidelines. The detailed list of labels can be found at <https://github.com/etcd-io/etcd/labels>.
 
 ## Scope
 
-These guidelines serves as a primary document for triaging an incoming issues in
-`etcd`. Everyone is welcome to help manage issues and PRs but the work and responsibilities discussed in this document are created with `etcd` maintainers and active contributors in mind.
+This document serves as the primary guidelines for triaging incoming issues in `etcd`.
+
+All contributors are encouraged and welcome to help manage issues which will help reduce burden on project maintainers, though the work and responsibilities discussed in this document are created with `etcd` project reviewers and members in mind as these individuals will have triage access to the etcd project which is a requirement for actions like applying labels or closing issues.
+
+Refer to [etcd community membership](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md) for guidance on becoming and etcd project member or reviewer.
+
+## Step 1 - Find an issue to triage
```

Comment:
idea: have a #triage-wanted label that maintainers can put to delegate triage to members. I assume maintainers skim through new incoming issues. This label will give signal boost. Ideally all issues should be triaged but I don't think that's happening.

Reply:
Hmm I see what you mean, though my thinking is that these guidelines are to help all us new members triage in a useful way with **full autonomy**, i.e. without needing maintainers to do anything at all to prod us.  I think we are already starting to do that.

Maybe we could re-evaluate in a month or two after these merge and if there are issues still going without triage for too long then we could add a way for maintainers to prod / suggest we triage?

Some other projects namely kubernetes also by default add labels for needing triage to all new issues, but I think that is less useful on etcd where we don't have any bot automation and there is less volume of issues. 

Curious what others think,  if more folks want to add it straight away I will update and add a section for it.